tests/unit/dataactbroker/test_file_generation_handler.py

```diff
@@ -62,8 +62,7 @@ def test_generate_d1_file_query(monkeypatch, mock_broker_config_paths, database,
             expected1.append(str(dap_one.__dict__[value]))
             expected2.append(str(dap_two.__dict__[value]))
 
-    assert file_rows[1] == expected1
-    assert file_rows[2] == expected2
+    assert expected1 in file_rows and expected2 in file_rows
```

Comment:
if we're doing an and, this should remain 2 assert statements so we can see where it breaks (if it does) more clearly